yarn-project/noir-protocol-circuits/src/crates/types/src/abis/public_circuit_public_inputs.nr

```diff
@@ -1,5 +1,5 @@
-use crate::contrakt::storage_read::StorageRead;
-use crate::contrakt::storage_update_request::StorageUpdateRequest;
+use crate::contract::storage_read::StorageRead;
```

Comment:
Hmm was this intentional to avoid the contract keyword?

Reply:
You were correct. Kev confirmed that it was intentional reverted the change. Thanks